lego/apps/comments/models.py

```diff
@@ -15,7 +15,7 @@ def get_queryset(self):
 
 class Comment(BasisModel):
 
-    text = ContentField(allow_images=False)
+    text = ContentField(allow_images=False, null=True)
```

Comment:
Have you tested this (both frontend and backend)?

Reply:
Tested frontend, but can write a test for backend as well.